packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.jsx

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const ContractTypeGlossary = ({ category }) => {
+    let ContractTypeGlossaryTemplate;
+    if (category) {
+        switch (category) {
+            case 'accumulator':
+                ContractTypeGlossaryTemplate = (
+                    <React.Fragment>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
```

Comment:
It's for future, not current implementation: as we are going to have glossary for all trade types, it'll be better to split them into different files. Like single fro Accumulator, single for Vanillas and etc. And in current file you just choose correct glossary component with the help of switch-case.
Again, it's for future, not current implementation, just to make it more manageable